front/src/applications/stdcm/utils/formatStdcmConfV2.ts

```diff
@@ -141,10 +143,19 @@ export const checkStdcmConf = (
         ...stepLocation
       } = step;
 
-      const secondary_code = 'trigram' in stepLocation || 'uic' in stepLocation ? ch : undefined;
+      const duration = stopFor ? sec2ms(ISO8601Duration2sec(stopFor) || Number(stopFor)) : 0;
+
+      if ('track' in stepLocation) {
+        return {
+          duration,
+          location: { track: stepLocation.track, offset: mToMm(stepLocation.offset) },
+        };
+      }
 
+      const secondary_code = 'trigram' in stepLocation || 'uic' in stepLocation ? ch : undefined;
       return {
-        duration: stopFor ? sec2ms(ISO8601Duration2sec(stopFor) || Number(stopFor)) : 0,
+        duration,
+        // TODO drop TS V1: we should store the offset in mm in the store
```

Comment:
Could you rename the comment like that to harmonize with existing, I'm affraid we might miss these :)

```suggestion
        // TODO DROP V1: we should store the offset in mm in the store
```